codegen/projections/high_score_service/lib/high_score_service/config.rb

```diff
@@ -31,7 +31,7 @@ module HighScoreService
   #   @option args [Hearth::HTTP::Client] :http_client (Hearth::HTTP::Client.new)
   #     The HTTP Client to use for request transport.
   #   @option args [Hearth::IdentityResolver] :http_login_identity_resolver
-  #     A Hearth::IdentityResolver that returns a Hearth::Identities::HTTPLogin for operations modeled with the smithy.api#httpBasicAuth auth scheme.
+  #     A Hearth::IdentityResolver that returns a Hearth::Identities::HTTPLogin for operations modeled with the smithy.api#httpDigestAuth auth scheme.
```

Comment:
http_login_identity_resolver. (HttpLogin)... is used for both smithy.api#httpBasicAuth and smithy.api#httpDigestAuth right?

When both are configured, this leads to maybe confusing documentation here.

Reply:
Yep it should say both

Reply:
I dont really see a good way around this though other than updating this documentation to be more generic (trying to dynamically combine them.... its tricky and special cased).

Reply:
Generic is fine .. "operations modeled to use it"